src/main/java/build/buildfarm/common/redis/RedisClient.java

```diff
@@ -148,11 +150,15 @@ public <T> T call(JedisContext<T> withJedis) throws IOException {
           status = Status.DEADLINE_EXCEEDED;
         } else if (cause instanceof IOException) {
           throw (IOException) cause;
+        } else if (cause instanceof JedisClusterOperationException) {
+          cause = cause.getSuppressed()[cause.getSuppressed().length - 1];
```

Comment:
This suppression extract is still in place - there are no suppressions for the non-JedisClusterMaxAttemptsException case when throwing.